File: 2021W01/JeomB/bj_4153.java
//https://www.acmicpc.net/source/25032956
import java.util.ArrayList;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        ArrayList<String> arr = new ArrayList<>();

        while (true) {
            double a = sc.nextInt();
            double b = sc.nextInt();
            double c = sc.nextInt();
            double result1,result2,result3;
            if (a==0&&b==0&&c==0) {
                break;
            }
            result1 = Math.pow(a, 2) + Math.pow(b, 2);
            result2 =  Math.pow(b, 2) + Math.pow(c, 2);
            result3 =  Math.pow(a, 2) + Math.pow(c, 2);
            if (result1 == Math.pow(c, 2)|result2== Math.pow(a, 2)|result3== Math.pow(b, 2)) {
                arr.add("right");
            } else {
                arr.add("wrong");
            }
        }
       for(int j=0;j<arr.size();j++){
           System.out.print(arr.get(j));
           if (arr.size()-1!=j){
               System.out.println();
           }
       }
            sc.close();
        }
    }

File: 2021W01/jjin/bj_4153.ts
// https://www.acmicpc.net/problem/4153
//import fs = require("fs");
//const input : string = fs.readFileSync("/dev/stdin").toString().trim();
const input : string = `6 8 10
25 52 60
5 13 12
0 0 0`;
const arr : string[] = input.split("\n");
arr.pop();
let str = "";
const sqm = (number) => Math.pow(number,2);
for(let i=0; i<arr.length; i++){
    const [a,b,c] : number[] = arr[i].trim().split(" ").map(elem=>sqm(+elem)).sort( (a,b)=>a-b);
    if(a+b===c)         str += "right\n";
    else                str += "wrong\n";;
}
console.log(str.trim())